python-pytest.el

```diff
@@ -429,11 +437,6 @@ When present ON-REPLACEMENT is substituted, else OFF-REPLACEMENT is appended."
           (format "%s %s" option it)))
    args))
 
-(defun python-pytest--choose-traceback-style (prompt _value)
```

Comment:
any technical reason this was moved up in the file? usually i prefer to have helpers down at the bottom and actual ‘entry points’ at the top.

Reply:
No reason. Can move the it back here (or inline as suggested in other comment).

Reply:
in that case please move it down here just to keep the groupings somewhat organised

Reply:
moving